README.md

```diff
@@ -1,6 +1,6 @@
 # Monti APM Core
 
-This is a fork of [kadira-core](https://github.com/kadirahq/kadira-core).
+This is a fork of [monti-apm-core](https://github.com/monti-apm/monti-apm-core).
```

Comment:
This is linking back to itself.

I'm not sure if we need to keep this line. We could move it further down in the readme.